build.gradle.kts

```diff
@@ -4,11 +4,13 @@ import java.net.URI
 buildscript {
     repositories {
         jcenter()
+        google()
```

Comment:
I think it has since been resolved, but for a while `google` needed to be first due to some weird dependency resolution issue.

Even though I think it's fixed, probably better to have `google` first, just to be safe, since it won't hurt anything (other than maybe extra dependency lookup misses).